Add instructions for pushing to GitHub from the VEDA Hub (#158)
NASA-IMPACT/veda-docs: notebooks/quickstarts/github-authentication.md

This is a quick how-to guide for pushing to GitHub from the VEDA JupyterHub.

## Configure the veda-hub-github-scoped-creds app

First, you need to install the GitHub auth application in the organization and repository that you will be pushing to. This only needs to be done once per repository.

1. Navigate to [https://github.com/apps/veda-hub-github-scoped-creds](https://github.com/apps/veda-hub-github-scoped-creds).
2. Select `Configure`.
3. Select the organization that contains the repository that you would like to push to.
4. Select either `All repositories` or `Only select repositories` depending on whether you only want to allow pushing to specific repositories. In order for a Hub user to write to those repositories, the repository needs to be added to the application **and** the user needs to have read/write permissions for the repository. The following restrictions define whether you can install the application yourself or if you need to request permission:
    - Organization owners can install GitHub Apps for all repositories.
    - Repository admins can install the application if they only grant access to repositories that they administer.

>     - If a requested does not have sufficient permissions, GitHub will send a notification to the organization owner requesting that they install the app.

Reviewer: ```suggestion
    - If you do not have sufficient permissions, GitHub will send a notification to the organization owner requesting that they install the app.
```
